### PR TITLE
feat: event modal

### PR DESCRIPTION
**Pull request by 01001110J**
# Description
This PR adds the modal to add a new event.

Rresolves [PADV-1761](https://agile-jira.pearson.com/browse/PADV-PADV-1761)

#### Before
after

#### After
image

### How to test
- Configure the flag `enable_instructor_calendar` in the site admin 18000 or override the content  of the request `http://localhost:18000/api/mfe_config/` to have the flag in `true`

- Go to '/instructors/:instructorUsername' if `enable_instructor_calendar` is `true` you should be able to see the calendar along with the `events/` request, if the flag is `false` you shouldn't see any of both.

### CODE REVIEW

File: src/features/Instructors/InstructorsDetailPage/index.jsx
Comment (AuraAlba) [inline src/features/Instructors/InstructorsDetailPage/index.jsx:1]: If a create an event and then change a page, in the get events I don't see the new event I created, did you mention that to Cesar?

**Review by AuraAlba — CHANGES_REQUESTED** [review]

File: src/features/Instructors/InstructorsDetailPage/index.jsx
Comment (01001110J) [reply src/features/Instructors/InstructorsDetailPage/index.jsx:1]: Can you provide a video or something for this, I did not have this issue:

https://github.com/user-attachments/assets/c77963d6-abff-47dd-ae24-f3ae84ce14dc

Comment (AuraAlba) [reply src/features/Instructors/InstructorsDetailPage/index.jsx:1]: events

**Comment by 01001110J** [comment]
Functionality no longer required.

**Pull request closed without merging**